src/amo/components/StaticPages/About/index.js

```diff
@@ -52,8 +62,8 @@ export class AboutBase extends React.Component {
           <ul>
             <li dangerouslySetInnerHTML={sanitizeHTML(blogText, ['a'])} />
             <li>
-              {i18n.gettext('We often post news, tips, and tricks to our Twitter account,')}&nbsp;
-              <a href={'http://twitter.com/mozamo'}>{i18n.gettext('mozamo')}</a>
+              {i18n.gettext('We often post news, tips, and tricks to our Twitter account, ')}&nbsp;
+              <a href={'http://twitter.com/mozamo'}>mozamo</a>
```

Comment:
This should be a substitution, rather than implicit concatenation, as a translation's word order could be different.